### DISCUSSION TITLE
Does CE no longer support msvc to execute code and output results?

### DISCUSSION THREAD

**Discussion by hewillk** [compiler-explorer/compiler-explorer#4669] [Q&A] [answered]
I recall that some time ago CE was able to support msvc to output execution results, but recently I found that msvc seems to no longer support output of execution results. 

I wonder if it is my local issue or is this expected behavior?  If this is the expected behavior, what is the reason for not supporting it anymore? thanks (https://godbolt.org/z/PdYhMPvbq)

![image](https://user-images.githubusercontent.com/67143766/215691024-c172a363-f1ea-40cc-a3cb-8709313520dd.png)

**Comment by mattgodbolt** [comment, marked as answer]
I'm afraid it no longer supports MSVC code execution. I should write a blog post explaining: I tweeted and Mastodon'd and posted the news on Slack and Discord, but right now unfortunately due to security concerns we've had to disable this feature.

We're working on a fix, though it could be a long while before it's back. Sorry for the inconvenience!
